nrf_rpc/nrf_rpc.c

```diff
@@ -102,6 +102,7 @@ struct internal_pool_data {
 static struct nrf_rpc_cmd_ctx cmd_ctx_pool[CONFIG_NRF_RPC_CMD_CTX_POOL_SIZE];
 
 static struct nrf_rpc_os_event groups_init_event;
+static struct nrf_rpc_os_event error_event;
```

Comment:
I would call it as `internal_data_consumed` and use it for all types of internal packets for consistency.

Reply:
Kind of agree, but let's do improvements as separate activity. I guess this PR is mostly hotfix.